include/nbl/asset/format/EFormat.h

```diff
@@ -1738,7 +1746,7 @@ inline value_type getFormatMinValue(E_FORMAT format, uint32_t channel)
 template <typename value_type>
 inline value_type getFormatPrecision(E_FORMAT format, uint32_t channel, value_type value)
 {
-    _NBL_DEBUG_BREAK_IF(isBlockCompressionFormat(format)); //????
+    //_NBL_DEBUG_BREAK_IF(isBlockCompressionFormat(format)); //????
```

Comment:
its hard to reason about the precision of a block compressed format.

This is because usually the compression is done by picking two end points in reduced prevision (RGB565) and then interpolating between them with reduced precision yet again.

So the "real" precision depends on the block's encoded values (to make this func work with block compressed, we'd need to pass a whole texel block data instead of channel's value)

Leave the assert here, never call this function for a block compressed format, and just point blank assume that the precision:
- is FLT_MAX for any channels not present
- 1.f for binary alpha formats (cut out transparency)
- highest it could possibly be (imagine whole tex block has the same value, in DXT this means 8bit SRGB prec, other block formats, no clue)

but don't implement that workaround here!

Reply:
oops, i didnt mean to commit that, i was just getting annoyed at the debug breaking there